.env.oss

```diff
@@ -1,85 +1,28 @@
-# OSS version of the .env file
+# OSS version of the .env.shared file
 # Note: The CLIENT_ID & CLIENT_SECRET are known keys for Artsy OSS
-# projects, and are not a problem. OSS people: Please don't abuse the keys,
+# projects, and are not a problem. OSS contributors: Please don't abuse the keys,
 # as then we'll have to change it, making it harder for others to learn from.
 # As such, these keys do not come under the Artsy security bounty either.
 
-WEBPACK_DEVTOOL=cheap-module-source-map
-
-# Shared
-API_REQUEST_TIMEOUT=10000
-API_URL=https://stagingapi.artsy.net
-APP_TIMEOUT=29s
-APP_URL=http://localhost:5000
-APPLICATION_NAME=
+# ------------------------
+# OSS Environment
+# ------------------------
 CLIENT_ID=e750db60ac506978fc70
 CLIENT_SECRET=3a33d2085cbd1176153f99781bbce7c6
-COOKIE_DOMAIN=
-DEFAULT_CACHE_TIME=
-ENABLE_SERVER_SIDE_CACHE=
-EMAIL_SIGNUP_IMAGES_ID=
-ERROR_PAGE_URL=
-FACEBOOK_ID=
-FACEBOOK_SECRET=
-GOOGLE_MAPS_API_KEY=
-LINKEDIN_KEY=
-LINKEDIN_SECRET=
-MAINTENANCE_PAGE_URL=
-MARKETING_SIGNUP_MODALS=
-NEW_RELIC_LOG_LEVEL=
-NODE_ENV=development
-OPENREDIS_URL=
-PAPERTRAIL_API_TOKEN=
+METAPHYSICS_ENDPOINT=https://metaphysics-staging.artsy.net
+API_URL=https://stagingapi.artsy.net
+POSITRON_URL=https://stagingwriter.artsy.net
+PREDICTION_URL=https://live-staging.artsy.net
+ADMIN_URL=http://admin-staging.artsy.net
+CMS_URL=http://cms-staging.artsy.net
+API_REQUEST_TIMEOUT=10000
+APP_TIMEOUT=29s
 PORT=5000
-POSITRON_URL=https://writer.artsy.net
-PREDICTION_URL=https://live.artsy.net
-PROFILE_MEMORY=
-S3_BUCKET=
-SAILTHRU_KEY=
-SAILTHRU_SECRET=
-SEGMENT_WRITE_KEY=
-SEGMENT_WRITE_KEY_SERVER=
-SENTRY_PRIVATE_DSN=
-SENTRY_PUBLIC_DSN=
-SESSION_COOKIE_KEY=
-SESSION_COOKIE_MAX_AGE=
-SHOW_ANALYTICS_CALLS=true
-STRIPE_PUBLISHABLE_KEY=
 WEBFONT_URL=http://webfonts.artsy.net
-VOLLEY_ENDPOINT=
-RECAPTCHA_KEY=
-
-# Desktop
-ADMIN_URL=http://admin.artsy.net
-CMS_URL=http://cms.artsy.net
-CONSIGNMENTS_APP_URL=https://sell.artsy.net/
-CONVECTION_APP_URL=
-CONVECTION_APP_ID=
-CRITEO_ARTWORKS_ACCOUNT_NUMBER=
-CRITEO_AUCTIONS_ACCOUNT_NUMBER=
-GALAXY_PUBLISHABLE_TOKEN=
-GALAXY_URL=https://galaxy-staging.artsy.net/
+CONSIGNMENTS_APP_URL=https://sell.artsy.net
```

Comment:
I think this app no longer exists, and can be removed along with [its references](https://github.com/artsy/force/search?q=CONSIGNMENTS_APP_URL).